pkg/daemon/ceph/client/upgrade.go

```diff
@@ -234,15 +234,21 @@ func LeastUptodateDaemonVersion(context *clusterd.Context, clusterInfo *ClusterI
 	if err != nil {
 		return vv, errors.Wrap(err, "failed to find daemon map entry")
 	}
+
+	vv = cephver.CephVersion{Major: 65535, Minor: 65535, Extra: 65535, Build: 65535, CommitID: ""}
```

Comment:
How about defining a const for 65535?

Reply:
I can, but would that help anything for maintainability or readability? It's just the max size of the data type, and any name I give it will be longer than 5 chars.

Reply:
Perhaps it could just be a local const such as `maxVer`, but it's just a nit and i'll approve if it's not worth it